force-app/main/default/classes/ContactsInLegacyHouseholds.cls-meta.xml

```diff
@@ -1,5 +1,15 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <ApexClass xmlns="http://soap.sforce.com/2006/04/metadata">
     <apiVersion>51.0</apiVersion>
+    <packageVersions>
+        <majorNumber>3</majorNumber>
+        <minorNumber>19</minorNumber>
+        <namespace>npe01</namespace>
+    </packageVersions>
+    <packageVersions>
+        <majorNumber>3</majorNumber>
+        <minorNumber>15</minorNumber>
+        <namespace>npo02</namespace>
+    </packageVersions>
     <status>Active</status>
```

Comment:
Is this an intended change to the ContactsInLegacyHouseholds meta file?

Reply:
No its not

Reply:
Reverted